lisp/gl.carp

```diff
@@ -1,6 +1,7 @@
 
-(if (osx?) (def *LIBNAME* "libglfw3.dylib") "")
-(if (linux?) (def *LIBNAME* "libglfw.so") "")
+;(if (osx?) (def *LIBNAME* "libglfw3.dylib") "")
+;(if (linux?) (def *LIBNAME* "libglfw.so") "")
+(def *LIBNAME* (str (if (windows?) "" "lib") "glfw" (:dylib-extension platform-specifics))
```

Comment:
I find this a lot harder to reason about.

Where is the '3' needed for 'libglfw3.dylib' now coming from?

I would much prefer an if for each os, as previously done.